zebra-rpc/src/server.rs

```diff
@@ -7,32 +7,27 @@
 //! See the full list of
 //! [Differences between JSON-RPC 1.0 and 2.0.](https://www.simple-is-better.org/rpc/#differences-between-1-0-and-2-0)
 
-use std::{fmt, panic};
```

Comment:
I think in general we do this ordering:

- std imports
- external imports
- zebra crates imports
- crate imports

Reply:
Addressed in [bbda07f](https://github.com/ZcashFoundation/zebra/pull/9459/commits/bbda07f13702120aa77c8265b8b5dc155d89bf97).